models/articles.js

```diff
@@ -29,4 +29,13 @@ exports.fetchAllArticles = () => {
     ON articles.article_id = comments.article_id GROUP BY articles.article_id`).then(({rows}) => {
         return rows;
     })
+}
+
+exports.fetchCommentsById = (id) => {
+    return db.query('SELECT * FROM comments WHERE article_id = $1', [id]).then((res) => {
+        if(!res.rowCount){
```

Comment:
Will need to change this to check if its empty because the ID isnt there, or empty because the id has no comments for it